gprofiler/profilers/python.py

```diff
@@ -211,7 +221,7 @@ def _profile_process(self, process: Process) -> ProfileData:
                     self._make_command(process.pid, local_output_path),
                     stop_event=self._stop_event,
                     timeout=self._duration + self._EXTRA_TIMEOUT,
-                    kill_signal=signal.SIGKILL,
+                    kill_signal=signal.SIGTERM if is_windows() else signal.SIGKILL,
```

Comment:
`SIGTERM` on windows does `TerminateProcess()` and kills it?

Reply:
SIGTERM does not exist on Windows

Reply:
Yeah I understand, I'm asking what's the logical substitute. Is it `SIGTERM`?

Reply:
I think SIGTERM is the closest substitute for Windows. Here's what's supported (https://docs.microsoft.com/en-us/previous-versions/xdkz3x12(v=vs.140)):

SIGABRT | Abnormal termination
SIGFPE | Floating-point error
SIGILL | Illegal instruction
SIGINT | CTRL+C signal
SIGSEGV | Illegal storage access
SIGTERM | Termination request